.github/workflows/run-code-quality-check.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           # https://megalinter.github.io/configuration/
           VALIDATE_ALL_CODEBASE: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }} # Validates all source when push on main, else just the git diff with main. Override with true if you always want to lint all sources
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          KOTLIN_KTLINT_ARGUMENTS: "--ruleset=custom-ktlint-rules/build/libs/custom-ktlint-rules.jar"
+          KOTLIN_KTLINT_ARGUMENTS: "--ruleset=com.twitter.compose.rules:ktlint:0.0.26"
```

Comment:
**bug**: You've removed our custom-ktlint-rules, you should keep using our custom rules.

Reply:
this is a draft as for now for testing, ill let you know once it is ready for review.